.configure

```diff
@@ -0,0 +1,15 @@
+{
+  "project_name": "simplenote-electron",
+  "branch": "trunk",
+  "pinned_hash": "bbd0130868d58b2e59c909433df83f4ae6721b76",
+  "files_to_copy": [
+    {
+      "file": "iOS/app_store_connect_fastlane_api_key.p8",
+      "destination": "~/.configure/simplenote-electron/secrets/app_store_connect_api_key.p8",
```

Comment:
Unfortunately, the existing notarization script requires the `p8` file and does not support reading the key content from memory like Fastlane does. As such, we need to track the encrypted `p8` in the repo.